README.md

```diff
@@ -13,5 +13,7 @@ A collection of projects supporting GCP integration
    * [Deployment Guide to Kubernetes (manual)](https://github.com/CrowdStrike/falcon-integration-gateway/tree/main/docs/chronicle)
  * Container Security
    * [Falcon Container Security Documents](container)
+ * VM Sensor Deployment
+  * [Google OS Config](osconfig)
```

Comment:
Should we say `VM Manager (OSConfig)` per Google documentation
![Google Results](https://github.com/CrowdStrike/Cloud-GCP/assets/41654718/965f5487-c83a-414e-b538-6fa301ac31fc)